lib/SimpleSAML/Store/Redis.php

```diff
@@ -31,13 +31,14 @@ public function __construct($redis = null)
             $host = $config->getString('store.redis.host', 'localhost');
             $port = $config->getInteger('store.redis.port', 6379);
             $prefix = $config->getString('store.redis.prefix', 'SimpleSAMLphp');
+            $password = $config->getString('store.redis.password', '');
 
             $redis = new \Predis\Client(
                 [
                     'scheme' => 'tcp',
                     'host' => $host,
                     'port' => $port,
-                ],
+                ] + (!empty($password) ? ['password' => $password] : []),
```

Comment:
Wow, I'm really surprised this is valid code..  It really feels like a comma is missing!